toolbox/+otp/RHS.m

```diff
@@ -1,36 +1,304 @@
 classdef RHS
+    % The right-hand side and related properties of a differential equation.
+    %
+    % This immutable class contains properties required by time integrators and other numerical methods to describe the
+    % following differential equation:
+    %
+    % $$M(t, y) y' = f(t, y)$$
+    %
+    % The mass matrix $M(t, y)$ is permitted to be singular in which case we have a differential-algebraic equation.
```

Comment:
It seems like most of the documentation is written without the use of the mathematical We, so maybe do passive voice?